resources/views/adm/mship/feedback/new.blade.php

```diff
@@ -0,0 +1,173 @@
+@extends('adm.layout')
+
+@section('scripts')
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-sortable/0.9.13/jquery-sortable-min.js"
+            integrity="sha384-mwD0+87SDVjJjyfTMQHNVV+IyWDM38MhzdCFZ+SRefmD75v+M5K0R3naFNLnZf1L"
+            crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-datetimepicker/2.5.4/build/jquery.datetimepicker.full.min.js" integrity="sha384-8Lb23xW0dVl+HHrv90JF6PpwygXa7Z0zZIK9+RWorNDyubrG7Ppu7JJw32U8op0i" crossorigin="anonymous"></script>
```

Comment:
The use of JS plugins such as datepicker still needs a bit more work due to the fact that they were previously used from AdminLTE, which we need to move to a more central location.

Using a CDN URL is fine for now, so feel free to stick with that until we can sort something better out